### ISSUE TITLE
OV.mutations has many rows with entries as column names

### ISSUE THREAD

**Issue by kieranrcampbell** [RTCGA/RTCGA#95]
### Actual behavior

There are 227 rows of OV.mutations where the entry is the column name, e.g. "bcr_patient_barcode" always has the value "bcr_patient_barcode"

### Steps to reproduce the problem

```{r}
library(RTCGA.mutations)
data("OV.mutations")
dplyr::filter(OV.mutations, Mutation_Status == "Mutation_Status")
```
begins with

```
  Hugo_Symbol Entrez_Gene_Id Center NCBI_Build Chromosome Start_position End_position
1 Hugo_Symbol Entrez_Gene_Id Center NCBI_Build Chromosome Start_position End_position
2 Hugo_Symbol Entrez_Gene_Id Center NCBI_Build Chromosome Start_position End_position
3 Hugo_Symbol Entrez_Gene_Id Center NCBI_Build Chromosome Start_position End_position
...
```

**Comment by MarcinKosinski** [comment]
@kieranrcampbell  what is your session info  -> what is your `RTCGA.mutations` version ? 
This is very old bug, and should be fixed in the newest release of `RTCGA.mutations`

check this

````{R
> library(RTCGA.mutations)
> library(dplyr)

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> data("OV.mutations")
> dplyr::filter(OV.mutations, Mutation_Status == "Mutation_Status")
 [1] Hugo_Symbol                   Entrez_Gene_Id                Center                       
 [4] NCBI_Build                    Chromosome                    Start_position               
 [7] End_position                  Strand                        Variant_Classification       
[10] Variant_Type                  Reference_Allele              Tumor_Seq_Allele1            
[13] Tumor_Seq_Allele2             dbSNP_RS                      dbSNP_Val_Status             
[16] Tumor_Sample_Barcode          Matched_Norm_Sample_Barcode   Match_Norm_Seq_Allele1       
[19] Match_Norm_Seq_Allele2        Tumor_Validation_Allele1      Tumor_Validation_Allele2     
[22] Match_Norm_Validation_Allele1 Match_Norm_Validation_Allele2 Verification_Status          
[25] Validation_Status             Mutation_Status               Sequencing_Phase             
[28] Sequence_Source               Validation_Method             Score                        
[31] BAM_file                      Sequencer                     dataset                      
[34] type                          classification                start                        
[37] end                           gene                          chr                          
[40] ref_allele                    tum_allele1                   tum_allele2                  
[43] context                       bcr_patient_barcode          
<0 rows> (or 0-length row.names)
> sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

locale:
 [1] LC_CTYPE=pl_PL.UTF-8       LC_NUMERIC=C               LC_TIME=pl_PL.UTF-8        LC_COLLATE=pl_PL.UTF-8    
 [5] LC_MONETARY=pl_PL.UTF-8    LC_MESSAGES=pl_PL.UTF-8    LC_PAPER=pl_PL.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=pl_PL.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_0.5.0                  RTCGA.mutations_20151101.2.2 RTCGA.methylation_1.0.2     
[4] RTCGA_1.3.4                  archivist_2.1.0.1001        

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.7      survminer_0.2.2  plyr_1.8.4       bitops_1.0-6     viridis_0.3.4    tools_3.3.1     
 [7] digest_0.6.10    lubridate_1.6.0  RSQLite_1.0.0    tibble_1.2       gtable_0.2.0     lattice_0.20-34 
[13] Matrix_1.2-7.1   DBI_0.5-1        gridExtra_2.2.1  httr_1.2.1       stringr_1.1.0    knitr_1.14      
[19] xml2_1.0.0       grid_3.3.1       data.table_1.9.6 R6_2.2.0         XML_3.98-1.4     survival_2.39-5 
[25] ggplot2_2.1.0    purrr_0.2.2      magrittr_1.5     scales_0.4.0     ggthemes_3.2.0   splines_3.3.1   
[31] assertthat_0.1   rvest_0.3.2      colorspace_1.2-6 stringi_1.1.2    RCurl_1.95-4.8   lazyeval_0.2.0  
[37] munsell_0.4.3    chron_2.3-47    

````

**Comment by MarcinKosinski** [comment]
fixed long time ago in #23
